packages/services/f-braze-adapter/src/services/utils/filterByEnabledCardTypes.js

```diff
@@ -1,3 +1,8 @@
 const filterByEnabledCardTypes = (cards, enabledCardTypes) => cards
-    .filter(({ type }) => (type ? enabledCardTypes.includes(type) : false));
+    .filter(({ type }) => {
+        if (enabledCardTypes.length > 0) {
+            return !type || enabledCardTypes.includes(type);
```

Comment:
If I'm reading this correctly, this will pass any card object that doesn't have a `type` field? Based on the previous condition I would have expected:
```suggestion
            return type && enabledCardTypes.includes(type);
```

Reply:
yes correct sorry, was doing this after brands. I will make the change